.github/workflows/ci.yml

```diff
@@ -27,6 +27,7 @@ jobs:
 
       - name: Lint code
         run: |
+          echo 'this is an update'
```

Comment:
What is this?

Reply:
A test for github ci update. I have seen Mcollina has new commit to delete this, and I will have a merge